docs/design/generics/details.md

```diff
@@ -2534,20 +2531,19 @@ constraint.
 > constraint is satisfied.
 
 A same-type constraint is written `where X == Y`, where `X` and `Y` both name
-facets. The constraint is that `X as type` must be the same as `Y as type`. It
-can be used only in the type expression position.
+facets. The constraint is that `X as type` must be the same as `Y as type`.
```

Comment:
Maybe say "It would normally only be used in the type expression position." here and below?